server/tests/api_v4/authenticated/test_events_listen.py

```diff
@@ -62,3 +62,54 @@ async def test_keep_alive(minimalorg: MinimalorgRpcClients, backend: Backend) ->
                 break
 
     assert got_keepalive
+
+
+async def test_last_event_id(minimalorg: MinimalorgRpcClients, backend: Backend) -> None:
+    async with minimalorg.alice.events_listen() as alice_sse:
+        raw_source = alice_sse._iter_events  # pyright: ignore[reportPrivateUsage]
+
+        for i in range(1, 4):
+            backend.event_bus._dispatch_incoming_event(  # pyright: ignore[reportPrivateUsage]
+                EventPinged(
+                    organization_id=minimalorg.organization_id,
+                    ping=f"event{i}",
+                )
+            )
+
+        # First event is always ServiceConfig
+        event = await alice_sse.next_event()
+        assert event == authenticated_cmds.v4.events_listen.RepOk(
+            authenticated_cmds.v4.events_listen.APIEventServerConfig(
+                active_users_limit=ActiveUsersLimit.NO_LIMIT,
+                user_profile_outsider_allowed=True,
+            )
+        )
+
+        event = await alice_sse.next_event()
```

Comment:
```suggestion
        # 2. Get the event_id of the first EventPinged 
        event = await alice_sse.next_event()
```

Reply:
We aren't getting the id of the first event